en_US/changes/breaking-changes-5.4.md

```diff
@@ -0,0 +1,16 @@
+# Breaking Changes
+
+## e5.4.0
+
```

Comment:
```suggestion
[#12201](https://github.com/emqx/emqx/pull/11994) Support hot update of TCP/SSL/WS/WSS MQTT listeners configuration.

 This allows changing most of the configuration parameters without restarting the listener and disconnecting the clients.

In case of TCP/SSL listeners, changes to the following parameters still require full listener restart and clients disconnect:
 * `bind`
 * `tcp_options.backlog`

In case of WS/WSS listeners, any parameter can be freely changed without losing the connected clients. However, changing transport related parameters will cause listening socket to be re-opened, namely:
 * `bind`
 * `tcp_options.*`
 * `ssl_options.*`
 
```

Reply:
@zmstone Do you need add this PR to the changes-ee-54.md file?

Reply:
right. updated.